### ISSUE TITLE
Requirement of Neo4j Drivers on boot

### ISSUE THREAD

**Issue by jodom** [stellasia/neomap#78]
At app start, we block render until we are sure we have an instance of neo4j driver. This blocks render indefinitely if driver is never resolved.

_See original discussion in  https://github.com/stellasia/neomap/pull/76#discussion_r484086056_

**Comment by jodom** [comment]
cc @stellasia, @wagnerjt for comments

**Comment by stellasia** [comment]
Project to follow, will probably be useful for us: https://github.com/adam-cowley/graphapp-starter-react

**Comment by jodom** [comment]
Yep 🙂. For starters, we really need this dev flow formalized for neomap :: [cid:77726d5e-69d9-4878-bb68-74c3b7affe70]

- Jodom
________________________________
